### DISCUSSION TITLE
xPoP development (Proof Of Payment) [JS/PY]

### DISCUSSION THREAD

**Discussion by wojake** [XRPLBounties/Proposals#118] [Ideas]
Code the use of xPoP on Javascript and Python and showcase its use case(s). The code must be clearly documented and a well written material must be included to explain the technicalities of how the protocol works.

Very little development and adoption has been made on xPoP, I believe it will be used in the near future and I want developers on the XRPL to have a technical understanding of it. I'd appreciate it if this program could fund this bounty, and I will not pursue this bounty.

# Resources
xPoP is a protocol which allows entities to verify on whether or not a transaction is validated by a set of nodes on a UNL on a particular network.

1. https://www.youtube.com/watch?v=ftrvdiWHpyk&ab_channel=XRPLedgerFoundation
2. https://www.youtube.com/watch?v=Z1KPqZKXe2E&ab_channel=XRPDEV
3. https://github.com/RichardAH/xpop-generator

**Comment by justinr1234** [comment]
Looking into this a bit..

**Comment by ckniffen** [comment]
Thank you for the bounty proposal. While Ripple has chosen not to fund this bounty at this time, the idea remains for other organizations to adopt and fund, they choose to do so. We appreciate your engagement with XRPL Bounties and look forward to considering any other proposals that you may have.
